deployment/overlays/samples/cert-manager/probes.yaml

```diff
@@ -0,0 +1,17 @@
+- op: add
+  path: /spec/template/spec/containers/0/livenessProbe/exec/command/-
+  value: "-tls"
+- op: add
+  path: /spec/template/spec/containers/0/livenessProbe/exec/command/-
+  value: "-tls-ca-cert=/etc/kubernetes/node-feature-discovery/certs/ca.crt"
+- op: add
+  path: /spec/template/spec/containers/0/livenessProbe/exec/command/-
+  value: "-tls-client-key=/etc/kubernetes/node-feature-discovery/certs/tls.key"
+- op: add
+  path: /spec/template/spec/containers/0/livenessProbe/exec/command/-
+  value: "-tls-client-cert=/etc/kubernetes/node-feature-discovery/certs/tls.crt"
+
+- op: replace
+  path: /spec/template/spec/containers/0/readinessProbe/exec/command
+  value: ["/usr/bin/grpc_health_probe", "-addr=:8080", "-tls", "-tls-ca-cert=/etc/kubernetes/node-feature-discovery/certs/ca.crt", "-tls-client-key=/etc/kubernetes/node-feature-discovery/certs/tls.key", "-tls-client-cert=/etc/kubernetes/node-feature-discovery/certs/tls.crt"]
```

Comment:
Could you change this similarly to liveness probe command above?

Reply:
done.